src/main/java/net/fabricmc/loom/task/service/TinyRemapperService.java

```diff
@@ -126,8 +126,8 @@ private static IMappingProvider gradleMixinMappingProvider(SharedServiceManager
 	// Set to true once remapping has started, once set no inputs can be read.
 	private boolean isRemapping = false;
 
-	public TinyRemapperService(List<IMappingProvider> mappings, boolean useMixinExtension, @Nullable KotlinClasspath kotlinClasspath) {
-		TinyRemapper.Builder builder = TinyRemapper.newRemapper();
+	public TinyRemapperService(List<IMappingProvider> mappings, boolean useMixinExtension, @Nullable KotlinClasspath kotlinClasspath, LoomGradleExtension extension) {
```

Comment:
I wouldnt pass the extension through like this, with the way things are moving in Gradle the idea is to move these services away from doing this. (kinda hard to explain here, see Graadle's docs on config caching.

I would change this to pass the `Set<String>` through the constructor